cli/src/main/java/org/jboss/as/cli/embedded/EmbedHostControllerHandler.java

```diff
@@ -258,9 +258,14 @@ public void cliEvent(CliEvent event, CommandContext ctx) {
     }
 
     private void configureLogContext(LogContext embeddedLogContext, File jbossHome, CommandContext ctx) {
-        File standaloneDir =  new File(jbossHome, "domain");
-        File configDir =  new File(standaloneDir, "configuration");
-        File logDir =  new File(standaloneDir, "log");
+
+        final String domainBaseDir = WildFlySecurityManager.getPropertyPrivileged("jboss.server.base.dir", jbossHome.getAbsolutePath() + File.separator + "domain");
+        final String domainConfigDir = WildFlySecurityManager.getPropertyPrivileged("jboss.server.configuration.dir", "configuration");
+        final String domainLogDir = WildFlySecurityManager.getPropertyPrivileged("jboss.server.log.dir", "log");
```

Comment:
The `domainConfigDir` and `domainLogDir` are never used. Depending on what we're doing too all these properties should probably be `jboss.domain.*`. The `domainBaseDir` and `domainConfigDir` definitely should be.